cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TopicOperator.java

```diff
@@ -96,7 +96,7 @@ protected TopicOperator(String namespace, String cluster, Labels labels) {
 
         super(namespace, cluster, labels);
         this.name = topicOperatorName(cluster);
-        this.replicas = TopicOperatorSpec.DEFAULT_REPLICAS;
+        this.replicas = io.strimzi.api.kafka.model.TopicOperatorSpec.DEFAULT_REPLICAS;
```

Comment:
Why the entire package is needed? Does the compiler complain about that?

Reply:
You cannot annotate an `import` stmt with `@SuppressWarnings("deprecation")`, so those warnings will be emitted and the build will fail. Work around is to not use `import` and fully qualify the class everywhere you need it.